app/routes/concept.ts

```diff
@@ -10,13 +10,31 @@ import type MetaDataService from 'codecrafters-frontend/services/meta-data';
 import RouterService from '@ember/routing/router-service';
 import Store from '@ember-data/store';
 import RouteInfoMetadata, { HelpscoutBeaconVisibility } from 'codecrafters-frontend/utils/route-info-metadata';
+import { tracked } from '@glimmer/tracking';
 
-export type ConceptRouteModel = {
+export class ConceptRouteModel {
   allConcepts: ConceptModel[];
   concept: ConceptModel;
   conceptGroup: ConceptGroupModel;
-  latestConceptEngagement: ConceptEngagementModel;
-};
+  @tracked latestConceptEngagement: ConceptEngagementModel;
```

Comment:
This is pretty non-standard - can we move the tracked property to be in the controller instead somehow?

Reply:
Yes, I tried that initially, it needs to be a controller property, and it needs to be set up using the `setupController` hook, I can do that, it just seemed a bit messier, more code than just changing `type` -> `class`

Reply:
I have reverted `ConceptRouteModel` back to being a `type`, and moved `latestConceptEngagement` property from the model into the `ConceptController`, where it is naturally `@tracked`.

I've also moved the logic of assigning `latestConceptEngagement` from the `model` hook into the `setupController` hook in the route.